PayGreen/paygreen-ui
# PGX-4340 - Amélioration du composant Card

## How to test ?

- Launch the storybook with `yarn storybook`
- Go to the section Card and check the example `Card with colored background`

_The new props works as follow : If colorBgTheme's value equal to null then the background color is white00, else the background color is one of the light varient of the four PG colors._

Initial commit for pgx-4340-card_upgrade
feat: New props for Cards -> colorBgTheme
feat: PropTypes added to Card for colorBgTheme
feat: New story to show the result of colorBgTheme
feat: Now, if colorBgTheme different from null, the background color of the card is the light version of the choosen color (primary, secondary, etc.)
